src/ui.c

```diff
@@ -215,7 +215,7 @@ void ui_update(void)
     }
 
     ui_fm_update();
-    SYS_doVBlankProcessEx(IMMEDIATLY);
+    SYS_doVBlankProcessEx(IMMEDIATELY);
```

Comment:
Unrelated typo

Reply:
I was curious where you got the low B value from, so I checked the original site you linked to.
It doesn't seem linear, as you can see from my quick check of their math:

1146-1081=65
1081-1021=60 (-5)
1021-964=57 (-3)
964-910=54 (-3)
910-858=52 (-2)
858-810=48 (-4)
810-765=45 (-3)
765-722=43 (-2)
722-681=41 (-2)
681-644=37 (-4)

And since it seems your 607 implies you subtracted 37 again, I think it may be closer to subtract 34 (a change of -3 from 37):
644-34=610

Sorry for nitpicking, but since I didn't see where your low B came from, I had to try calculating it myself.

I have been unable to build the ROM myself, when last attempted, so I'm unable to test your changes.  I own an NTSC Sega Genesis, but I don't own a Mega Drive to compare with, even if I could build it.  If you have any hints on compiling it on GNU/Linux I wouldn't mind hearing them.

Thanks,

  Lucas